docs/resources/recognition.md

```diff
@@ -0,0 +1,54 @@
+# Recognizing Contributions
+
+There are a variety of ways that we would like to recognize community contributions to hackweeks. 
+
+## Website Presence
+
+We invite all people who join us in organizing and hosting a hackweek to be included in our "Meet the team" section of each hackweek website. 
+
+We use this YAML markdown template for each team member. 
+
+```
+- title: 
+  avatar: 
+  role: 
+  organizations:
+  - name: 
+    url: 
+  - name: 
+  bio: 
+  expertise:
+  - 
+  - 
+  social:
+  - icon: github
+    icon_pack: fab
+    link: 
+  user_groups:
+  - 
+  ```
+ Here is an example of how the text you include in this template gets rendered on the website:
+
+  ```{image} ../images/website-bio.jpg
+:alt: example of how to fill out website bio
+:width: 1100px
+:align: center
+```
+
+**Notes:**
+- You must precisely follow the indentation and spacing layout shown here, so that the website correctly interprets the YAML format.
+- We encourage everyone to use their GitHub avatar as the source of their personal photograph.
+- Only your GitHub username (not a full url) is required for the `link` key under `social`.
+- Acceptable user groups (roles) can be found [here](https://uwhackweek.github.io/hackweeks-as-a-service/services/index.html).
+
+### Uploading to the website
+
+To include your content on the website, please make a pull request to the `team.yaml` file located in the website for the event you're involved with. For example, here is the [file for the ICESat-2 2022 hackweek](https://github.com/ICESAT-2HackWeek/website2022/blob/main/book/team.yaml). 
+
+## Online Tutorials
+
+All of our tutorials are recorded and shared on the [eScience Institute's youtube channel](https://www.youtube.com/channel/UCLKSeDZWDqBzKRfHTabg1lQ). We encourage tutorial leads to link to these videos from their social media accounts and websites as a way to demonstrate their teaching competencies.

```

Comment:
Shall this link be changed to the 'Playlists'?
There we have the videos nicely organized by Hackweeks and might be an easier entry to find past content
https://www.youtube.com/c/UWeScienceInstitute/playlists